### ISSUE TITLE
When useTimeZoneWithRRules is true and having a weekly occurrence something goes wrong

### ISSUE THREAD

**Issue by machfr** [u01jmg3/ics-parser#233]
Hey
I have an issue when I set useTimeZoneWithRRules to true and I have a occurance weekly 

```
ical = new ICal($response, array(
    'shouldFilterByWindow'		=> true,
    'windowMinTimestamp'		=> true,
    'filterDaysBefore'			=> 14,
    'filterDaysAfter'			=> 365,
    'replaceWindowsTimeZoneIds' => true,
    'useTimeZoneWithRRules'     => true,
));
```

```
BEGIN:VEVENT
DTSTART;TZID=Europe/Copenhagen:20190807T100000
DTEND;TZID=Europe/Copenhagen:20190807T113000
RRULE:FREQ=WEEKLY;BYDAY=WE
DTSTAMP:20190806T082820Z
UID:0ravk3521qkp7mbiffhlm0ab47@google.com
CREATED:20190806T082448Z
DESCRIPTION:
LAST-MODIFIED:20190806T082448Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:07-08-2019 - hver onsdag
TRANSP:OPAQUE
END:VEVENT
```

Then the first day aka. the 7th of august it will give us an event from 10:00 - 11:30 as it should, but at the same day it will show give us an event from 11:00 - 12:30 and all the next occurrences weekly will be from 11:00 - 12:30

So there is two issues here
1. On the first day it will give us 2 events.
1. On all the next occurrences it will show an hours wrong.

**Comment by u01jmg3** [comment]
Thanks for logging this issue.

Please bear with me. @s0600204 has kindly submitted PR #232 which I need to find time to review. This may fix your issue so feel free to test out the code.

If not, I am happy to accept a PR.

**Comment by machfr** [comment]
Does not seem to have fixed the issue

**Comment by s0600204** [comment]
Looking into this, it seems to happen only if the timezone of the php environment is not UTC/GMT **and** `useTimeZoneWithRRules` is `true`. Setting php's timezone (via `date_default_timezone_set()`) to "UTC" **or** passing `useTimeZoneWithRRules` as `false`, appears to rectify the problem.

----

The aforementioned PR happens to remove the `useTimeZoneWithRRules` parameter entirely. So yeah: that also fixes the issue.

**Comment by machfr** [comment]
Seems like forcing it to UTC is working (I tried to set it to Europe/London which did not), will test some more with it

**Comment by u01jmg3** [comment]
Closed by #232

**Comment by s0600204** [comment]
(`Europe/London` alternates between UTC and BST ("British Summer Time"; UTC+1) depending on the time of year. So in this case the recurrences that fall in winter will most likely have the correct time, those that fall in summer will not.)
